python/pyspark/sql/dataframe.py

```diff
@@ -861,11 +861,11 @@ def groupBy(self, *cols):
             Each element should be a column name (string) or an expression (:class:`Column`).
 
         >>> df.groupBy().avg().collect()
-        [Row(avg(age)=3.5)]
+        [Row(avg_c1=3.5)]
```

Comment:
system generated alias name is just for display, if we are going to write data out, users should manually alias it with a more formal name.